### DISCUSSION TITLE
Version 3.0.0 is Awesome

### DISCUSSION THREAD

**Discussion by JohannPopper** [johnfactotum/foliate#1108] [General]
What an incredible update. Foliate is now, by far, the best epub/ebook reader in existence. What an astonishing donation to the common good of humanity. I'm buying you several coffees and some donuts too, unless you're a bagel guy. And everyone else should do so too.

**Comment by johnfactotum** [comment]
> What an astonishing donation to the common good of humanity.

What an astonishing compliment. Thank you!

**Comment by polesowa** [comment]
Completely agree! John has done an amazing job and reading books is such a joy now! I would love to donate crypto if John has a way of letting fans send to him.

The only thing I can think to make Foliate better is the ability to organize my ebooks into categories. As an example, there is a flathub app called "NewsFlash" for RSS that allows me to organize those feeds into categories.
